src/coord/timestamp.rs

```diff
@@ -44,6 +45,9 @@ use crate::coord;
 use futures::executor::block_on;
 use itertools::Itertools;
 
+/// Number of seconds that must elapse before we update watermarks again
+const WATERMARK_UPDATE_INTERVAL: u64 = 1;
```

Comment:
Why not have this be a `Duration` ?

(I'm sure you know more about Rust time-related idioms, so do whatever you think is best, I'm just curious).

Reply:
Partially because I forgot that `from_secs` is now a `const` function, but also we really don't need to be checking the nanoseconds for this.